.buildkite/pipeline.yml

```diff
@@ -145,29 +94,61 @@ steps:
             - ci:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-base-${BRANCH_NAME}
             - ci:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-base
 
-  - label: 'Lint'
+  - label: "Lint"
     if: build.env("BUILD_RN_WITH_LATEST_NATIVES") != "true"
-    depends_on: 'ci-image'
+    depends_on: "ci-image"
     timeout_in_minutes: 10
     plugins:
       docker-compose#v3.9.0:
         run: ci
-    command: 'npm run test:lint'
+    command: "npm run test:lint"
 
-  - label: 'Unit tests'
+  - label: "Unit tests"
     if: build.env("BUILD_RN_WITH_LATEST_NATIVES") != "true"
-    depends_on: 'ci-image'
+    depends_on: "ci-image"
     timeout_in_minutes: 10
     plugins:
       docker-compose#v3.9.0:
         run: ci
-    command: 'npm run test:unit'
+    command: "npm run test:unit"
 
-  - label: 'Type checks/tests'
+  - label: "Type checks/tests"
     if: build.env("BUILD_RN_WITH_LATEST_NATIVES") != "true"
-    depends_on: 'ci-image'
+    depends_on: "ci-image"
     timeout_in_minutes: 10
     plugins:
       docker-compose#v3.9.0:
         run: ci
-    command: 'npm run test:types'
+    command: "npm run test:types"
+
+  - label: ":large_blue_circle: :large_blue_circle: :large_blue_circle: BROWSER STEPS :large_blue_circle: :large_blue_circle: :large_blue_circle:"
+    if: build.env("BUILD_RN_WITH_LATEST_NATIVES") != "true"
+    depends_on: "package-js"
+    commands:
+      - buildkite-agent pipeline upload .buildkite/browser-pipeline.yml
```

Comment:
Would be happy to see stages grouped using buildkite rather than our own workaround

Reply:
Talked in IRL to confirm it was the `:large_blue_circle:` we see as a workaround (I agree).  Unfortunately though Buildkite doesn't currently offer a better solution.  However, I will implement a group for each pipeline to see what we think (I think it'll probably be a good thing).

Reply:
Groups look good to me and definitely better than the 🔵🔵🔵 markers — I'm guessing those steps will always have to be visible though as they're what uploads the pipelines?

It does seems a bit weird to me that the number in the group is (presumably) the number of finished tests instead of "passed/total" but that's easy to get used to 🤷

Reply:
A step of some kind would be needed, but the horrible blue circles could I think be moved into the groups themselves to make the initial banner more readable.  I'd quite like to keep some kind of section separator in, as I generally use the body of the build page rather than the header.  But admittedly I have a different perspective to most and perhaps I need to look at changing my habit a little.

This PR certainly doesn't need to be the end of the discussion though, I'll continue to try out different approaches - this was very much just the first iteration.  Shrinking the 7 pipeline pages down to 2 was the main thing to achieve here.